rcos_io/services/github.py

```diff
@@ -111,3 +112,111 @@ class CommitInfo(TypedDict):
     commit: Commit
     author: User
     committer: User
+
+
+#    html url structure: https://github.com/Apexal/rcos_io
+#     api url structure: https://api.github.com/repos/Apexal/rcos_io
+
+#          get branches: https://api.github.com/repos/Apexal/rcos_io/branches
+#           get commits: https://api.github.com/repos/Apexal/rcos_io/commits
+#   get commits by user: ?author=username or email
+#     get commits since: ?since=ISO-8601 timestamp <-- starttime
+#     get commits until: ?until=ISO-8601 timestamp <-- endtime
+# get commits from head: ?sha=commit sha
+
+
+def gen_params(**kwargs):
+    """
+    generates params for request querystring
```

Comment:
Can you standardize the docstrings to follow the convention that we've been using, i.e. proper capitalization

Reply:
done